components/31-molecules/teaser/teaser--teaser--wide.twig

```diff
@@ -1,19 +1,20 @@
+{% if tag %}
+  {% set myTags = [{text: tag, type: 'span-tag'}] %}
+{% endif %}
+{% if tag2 %}
+  {% set myTags = myTags|merge([{text: tag2, type: 'span-tag'}]) %}
+{% endif %}
```

Comment:
Heeft tag2 een functie? Zoja welke componenten geven deze value door?

Reply:
Is legacy, would have preferred an array myself. 
But I was lazy and didn't want to rewrite other components.

Reply:
Refactored tag & tag2 to tags array.